lib/puppet/application/apply.rb

```diff
@@ -64,10 +64,10 @@ def help
       individual manifests.
 
       When provided with a modulepath, via command line or config file, puppet
-      apply can effectively mimic the catalog that would be served by puppet
-      master with access to the same modules, although there are some subtle
+      apply can effectively mimic the catalog that would be served by OpenVox
+      server with access to the same modules, although there are some subtle
       differences. When combined with scheduling and an automated system for
-      pushing manifests, this can be used to implement a serverless Puppet
+      pushing manifests, this can be used to implement a serverless OpenVox
```

Comment:
is OpenVox even required on this line (it's well implied, and would limit our long term maintenance of the page)